### ISSUE TITLE
ziplora training objective

### ISSUE THREAD

**Issue by bryanhpchiang** [mkshing/ziplora-pytorch#29]
in the code: https://github.com/mkshing/ziplora-pytorch/blob/main/train_dreambooth_ziplora_sdxl.py#L1500

<img width="521" alt="Image" src="https://github.com/user-attachments/assets/33eded6e-2380-4214-871e-9564bf673963" />

seems like we're training in a manner to similar to regular dreambooth / diffusion

but the diagram from the paper seems to imply passing in a text caption and computing mse loss on the generated images? (aka actually doing the sampling...)

not sure if that makes sense. it just seems kinda different

<img width="968" alt="Image" src="https://github.com/user-attachments/assets/3ac9f6f9-0d92-4575-83ca-2515a4cb1f0e" />
